### DISCUSSION TITLE
Possible Globe View

### DISCUSSION THREAD

**Discussion by Jayson-Tolleson** [maplibre/maplibre-gl-js#2352] [Q&A]
Hey all, I am Jayson Tolleson:
Hopefully I will be able to update this some how to include globeview.....
I would like to share my code some:

<!DOCTYPE html>
<html>
    <head>
        <title>MapLibreGL</title>
<script src="https://unpkg.com/deck.gl@latest/dist.min.js"></script>
<script src="https://unpkg.com/@deck.gl/geo-layers@8.9.7/dist.min.js"></script>
<script src="https://unpkg.com/@deck.gl/mapbox@8.9.7/dist.min.js"></script>
<script src="https://unpkg.com/@deck.gl/core@8.9.7/dist.min.js"></script>
<script src="https://unpkg.com/@deck.gl/layers@8.9.7/dist.min.js"></script>
<script src="https://unpkg.com/@deck.gl/mesh-layers@8.9.7/dist.min.js"></script>
<script src="https://unpkg.com/@deck.gl/extensions@8.9.7/dist.min.js"></script>
<script src="https://unpkg.com/maplibre-gl@2.4.0/dist/maplibre-gl.js"></script>
<script src="https://unpkg.com/luma.gl@7.3.2/dist/es5/index.js"></script>
<script src="https://unpkg.com/@luma.gl/constants@8.5.19/dist/es5/index.js"></script>
<script src="https://unpkg.com/@luma.gl/core@8.5.19/dist/es5/index.js"></script>
<script src="https://unpkg.com/@loaders.gl/core@3.3.2/dist/es5/index.js"></script>
<script src="https://unpkg.com/gl-matrix@3.4.3/cjs/index.js"></script>
<script src="https://unpkg.com/null@2.0.0/index.js"></script>
<script src="https://unpkg.com/ts-node@10.9.1/dist/index.js"></script>
<script src="https://unpkg.com/tsc@2.0.4/bin/tsc.js"></script>
<script src="https://unpkg.com/typescript@5.0.3/lib/typescript.js"></script>
<link href="https://unpkg.com/maplibre-gl@2.4.0/dist/maplibre-gl.css" rel="stylesheet" />
    </head>
    <body>
<div id="map" style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;    opacity: 1;"></div>
<script>

const where = {
  zoom: 12.11636800831692,
  pitch: 61.03929753705569,
  longitude: -117.429707050949082,
  latitude: 33.81651393996542,
  maxPitch: 85,
  bearing: 44.59239340534395,
  altitude: 1.5,
};

let mapLibre = new maplibregl.Map({
 ...where,
  container: "map",
  center: [where.longitude, where.latitude],
  dragRotate: true,
  style: {
    version: 8,
    sources: {
      sat: {
        type: "raster",
        tiles: ["https://mt0.google.com/vt/lyrs=s&hl=en&x={x}&y={y}&z={z}"],
        tileSize: 256,
        maxzoom: 12
      },
      terrainSource: {
        type: "raster-dem",
        tiles: [
          "https://s3.amazonaws.com/elevation-tiles-prod/terrarium/{z}/{x}/{y}.png"
        ],
        encoding: "terrarium",
        tileSize: 256,
        maxzoom: 12
      }
    },

    // TOGGLE:  MapLibre Terrain

     layers: [{ id: 'sat', type: 'raster', source: 'sat' }],
     terrain: { source: 'terrainSource', exaggeration: 2, elevationOffset: 0 },

  }
});
mapLibre.addControl(
new maplibregl.NavigationControl({
visualizePitch: true,
showZoom: true,
showCompass: true
})
);

</script>

    </body>
</html>

any help would be greatly appreciated....the goal is to have a fishing data map here....

**Comment by HarelM** [comment]
I would recommend sharing a jsbin or codepen instead of sharing the code this way... Also, it's not clear if you would like someone to help you in creating a globeview or do you have an initial solution that you would like others to help with improving.
